pkg/clients/handlers.go

```diff
@@ -10,39 +10,35 @@ import (
 	"strconv"
 
 	"github.com/go-chi/chi/v5"
-	"github.com/go-playground/validator/v10"
 
 	"github.com/canonical/identity-platform-admin-ui/internal/http/types"
 	"github.com/canonical/identity-platform-admin-ui/internal/logging"
 	"github.com/canonical/identity-platform-admin-ui/internal/validation"
 )
 
 type API struct {
-	service   ServiceInterface
-	validator *validator.Validate
+	apiKey           string
+	service          ServiceInterface
+	payloadValidator validation.PayloadValidatorInterface
 
 	logger logging.LoggerInterface
 }
 
 func (a *API) RegisterEndpoints(mux *chi.Mux) {
 	mux.Get("/api/v0/clients", a.ListClients)
 	mux.Post("/api/v0/clients", a.CreateClient)
-	mux.Get("/api/v0/clients/{id:.+}", a.GetClient)
-	mux.Put("/api/v0/clients/{id:.+}", a.UpdateClient)
-	mux.Delete("/api/v0/clients/{id:.+}", a.DeleteClient)
+	mux.Get("/api/v0/clients/{id}", a.GetClient)
+	mux.Put("/api/v0/clients/{id}", a.UpdateClient)
+	mux.Delete("/api/v0/clients/{id}", a.DeleteClient)
```

Comment:
```suggestion
	mux.Get("/api/v0/clients/{id:.+}", a.GetClient)
	mux.Put("/api/v0/clients/{id:.+}", a.UpdateClient)
	mux.Delete("/api/v0/clients/{id:.+}", a.DeleteClient)
```

rebase went wrong here

Reply:
Most likely :D